lib/net.c

```diff
@@ -67,9 +62,18 @@ close_num_ports(uint8_t nb_ports)
 	}
 }
 
+struct net_config *
+alloc_net_conf(void)
+{
+	config.gatekeeper_pktmbuf_pool = (struct rte_mempool **) \
```

Comment:
For the ``struct net_config``, since we integrate the ``struct rte_mempool      *gatekeeper_pktmbuf_pool[GATEKEEPER_MAX_NUMA_NODES]`` to it. However, the constant cannot be claimed in the LuaJIT. So, if we keep it as an array, we may need to use enum type for the constants. I think making it dynamically allocated is easier to manage.